apps/pg_subscriptor/lib/column.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Column do
+
+  @type kind :: ?n | ?u | ?t | ?b
+  @type t :: %__MODULE__{
+    kind: kind,
+    value: nil | binary | String.t(),
+  }
+
+  defstruct [:kind, :value]
+
+  def get_cols(0, data) do
+    {[], data}
+  end
+
+  def get_cols(num_of_cols, data) do
+    <<col_type::utf8, rest::binary>> = data
+
+    {value, rest} =
+      case col_type do
+        ?b ->
```

Comment:
What is the `?b`? I would expect to match the `col_type` as a simple string. Like so: `"b"`.

Reply:
Yep I agree that we should handle it as `col_tupe::8`. Postgres specifies that it is a single Byte so it does not make sense for it to be in utf-8 format (which can be 2 bytes as well).